lib/src/connected_device_operation.dart

```diff
@@ -43,7 +43,7 @@ class ConnectedDeviceOperation {
 
   Stream<List<int>> subscribeToCharacteristic(
     QualifiedCharacteristic characteristic,
-    Future<ConnectionStateUpdate> shouldTerminate,
+    Future<void> isDisconnected,
```

Comment:
A cosmetic change. This naming makes it much clearer, _why_ to stop the subscription when this future complete.

Reply:
much better indeed